docs/docs/tutorial/index.md

```diff
@@ -631,30 +582,24 @@ cardano-cli transaction sign \
   --out-file tx-signed.json
 ```
 
-To submit the transaction we can use our websocket session again. This command
-will print the `NewTx` command to copy paste into an already open websocket
-connection:
+Submit the transaction through the already open WebSocket connection. Generate the `NewTx` command for WebSocket submission:
 
 ```shell
 cat tx-signed.json | jq -c '{tag: "NewTx", transaction: .}'
 ```
 
 The transation will be validated by both `hydra-node`s and either result in a
 `TxInvalid` message with a reason, or a `TxValid` message and a
-`SnapshotConfirmed` with the new UTxO available in the head shortly after.
+`SnapshotConfirmed` with the new UTXO available in the head shortly after.
 
 🎉 Congratulations, you just processed your first Cardano transaction off-chain
 in a Hydra head!
 
-At this stage you can continue experimenting with constructing & submitting
-transactions to the head as you wish. Proceed in the tutorial once you're done
-and want to realize the exchanged funds from the Hydra head back to the Cardano
-layer one.
+Feel free to continue experimenting with additional transactions. Move on to the next steps when ready to finalize your activities within the head.
 
 ## Step 6: Closing the Hydra head
 
-Each participant of the head can close it at any point in time. To do this, we
-can use the websocket API and submit this command:
+Any participant can initiate closing the Hydra head. Use the WebSocket API to submit the closing command:
 
 ```json title="Websocket API"
 { "tag": "Close" }
```

Comment:
Please update the copy below thus:

The `hydra-node` will then submit a protocol transaction with the last known snapshot to the Cardano network. A smart contract on layer 1 will check the snapshot signatures and confirm the head is closed. The WebSocket API sends a `HeadIsClosed` message when this' Close' transaction is observed. Note that this can also happen if any other `hydra-node` closes the head.

:::caution Known bug

You might need to submit the `Close` command multiple times if the head is not getting closed within approximately 30 seconds.

See [#1039](https://github.com/input-output-hk/hydra/issues/1039) for details.

:::

The message will include a `contestationDeadline`, set using the configurable `--contestation-period`. Until this deadline, the closing snapshot can be contested with a more recent, multi-signed snapshot. Your `hydra-node` would contest automatically for you if the closed snapshot is not the last known one.'

Reply:
Updated